couch/tox.ini

```diff
@@ -2,7 +2,7 @@
 minversion = 2.0
 basepython = py38
 envlist =
-    py{27,38}-{1.6,2.3}
+    py{27,38}-{1.6,2.3,3.1}
```

Comment:
Seems latest supported version is 2.3.1, we can remove 1.6 from e2e testing asumming we have at least one unit test for the v1 codepath

Reply:
There are currently many integration tests but almost no unit test. Just one checking the configuration.

Also Couch v1 is a completely separate implementation while v2 and v3 share the same one. I believe it could be worth keeping it